warzone/utils/reverse_words.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+
+def reverse_words(value):
+    """
+    Creates a mirror(reverse) the words respecting the punctuation.
+
+    input = 'I am a software engineer. But,, I am also an operations engi.neer.'
+    expected = 'I ma a erawtfos reenigne. tuB,, I ma osla na snoitarepo igne.reen.'
+    """
+    match = re.finditer('[a-zA-Z]+', value)
+
+    for item in match:
+        value = value[:item.start()] + item.group()[::-1] + value[item.end():]
```

Comment:
Maybe using f-strings is more 'modern' ? :)

Reply:
I did try modernize it with f-strings but I don't found a solution that allow us to load the variabeles and susing slicing togheter. Please can you show an example of it?

Reply:
`value = f'{value[:item.start()]}{item.group()[::-1]}{value[item.end():]}'`